app/assets/javascripts/code/services/_sensors.js

```diff
@@ -142,15 +143,13 @@ export const sensors = (params, $http) => {
   return new Sensors();
 };
 
-const hasChangedToAll = newValue => !newValue;
-
 const max = (valueOf, xs) => {
   const reducer = (acc, x) => valueOf(x) > valueOf(acc) ? x : acc;
   return xs.reduce(reducer, xs[0]);
 };
 
 export const findAvailableSensorsForParameter = (sort, sensors, parameter) =>
-  parameter ?
+  parameter.id !== ALL_PARAMETER.id ?
```

Comment:
I'm always having troubles with reading negation in `ifs`. What do you think about doing `parameter.id == ALL_PARAMETER.id` and switching the order of lines 153, 154?

Reply:
Great point